build/charts/antrea/templates/_helpers.tpl

```diff
@@ -21,3 +21,11 @@
 {{- define "antreaImage" -}}
 {{- print .Values.image.repository ":" (include "antreaImageTag" .) -}}
 {{- end -}}
+
+{{- define "validateValues" -}}
+{{- if .Values.whereabouts -}}
```

Comment:
Do not know if there is a better way.

Reply:
I think you can do `{{- if (.Values.whereabouts).enable -}}`

If `.Values.whereabouts` is undefined, it will evaluate to false right away